maven_iuvs/fits_processing.py

```diff
@@ -0,0 +1,273 @@
+import os as _os
+import datetime
+import numpy as np
+from pathlib import Path
+import re 
+
+from maven_iuvs.search import find_files, get_latest_files
+
+
+def get_binning_scheme(hdul):
```

Comment:
most of the routines in this file probably belong in `binning.py', 'fits_processing' seems too generic... misfits can go into misc

Reply:
Fixed by 6467166340944ca6a69510b1a6725e519a1a501e